jni/mupen64plus-video-gliden64/src/OpenGL.cpp

```diff
@@ -1336,6 +1336,10 @@ void OGLRender::_initData()
 	for (u32 i = 0; i < VERTBUFF_SIZE; ++i)
 		triangles.vertices[i].w = 1.0f;
 	triangles.num = 0;
+
+#ifdef ANDROID
+    __android_log_write(ANDROID_LOG_DEBUG, "GLideN64", "Finish render initialization.\n");
+#endif
 }
```

Comment:
@fzurita Just an FYI, this change will be overwritten next time we sync with upstream.  If you want this to be permanent, please submit a pull request for this change to @Gonetz.